code/modules/power/lighting/light.dm

```diff
@@ -58,6 +58,10 @@
 	var/emergency_mode = FALSE
 	///If true, this light cannot ever have an emergency mode
 	var/no_emergency = FALSE
+	//If true, overrides lights to use emergency lighting (used in cascades)
+	var/emergency_cascade = FALSE
+	//Multiplier for this light's base brightness during a cascade
+	var/bulb_emergency_cascade_brightness_mul = 0.5
```

Comment:
Probably would have to have more vars associated to color, range and such that act as an "override". Or maybe override the default color, range and brightness vars and such in a throwaway removable `set_emergency_lighting` proc (like `set_cascade_lighting`), but I feel kind of mixed on that. I can technically "permanently set this because it's the end of the round", however it screams crap code.